gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-ui.yaml

```diff
@@ -11,7 +11,7 @@ info:
     version: 2.0.0
 
 servers:
-    - url: "{protocol}://{managementAPIHost}/management/v2"
+    - url: "{protocol}://{managementAPIHost}/management/v2/environments/{envId}"
```

Comment:
A server without environment id and no authentication is necessary for the `/bootstrap` resource for Console.